toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/Extensions.kt

```diff
@@ -82,13 +82,13 @@ public fun Activity.launchCustomTabs(pendingSignIn: OAuthUserSignIn?): Unit {
  * @since 200.8.0
```

Comment:
I'm assuming we would also want to update the since tag for the public `launchCustomTabs(..)`? You can dismiss this if it feels unnecessary.

```suggestion
 * @since 200.8.1
```